public/app/core/components/Login/LoginLayout.tsx

```diff
@@ -29,6 +29,7 @@ export const LoginLayout = ({ children, branding, isChangingPassword }: React.Pr
   const loginTitle = branding?.loginTitle ?? Branding.LoginTitle;
   const loginBoxBackground = branding?.loginBoxBackground || Branding.LoginBoxBackground();
   const loginLogo = branding?.loginLogo;
+  const hideEdition = branding?.hideEdition ?? Branding.HideEdition;
```

Comment:
Why do we need the `Branding.HideEdition` part? I think it can be read directly from setting or am I missing something?

Reply:
When I test this locally `branding` is `undefined`, so I just copied what the other ones do with the `Branding` which is set on the enterprise side...